site/src/component/LoadingSpinner/LoadingSpinner.scss

```diff
@@ -1,5 +1,5 @@
-.requirements-loading {
+.loading-spinner {
   display: flex;
   justify-content: center;
-  margin: 8px auto;
+  margin: 20px;
```

Comment:
`8px auto` for now, since its usage in visually smaller components (i.e. program requirements) will suffer more from bigger changes to the margin.

You can also override this in other places in their respective style sheets, i.e. `.planner > .loading-spinner { margin-block: 20px }` as appropriate